assets/css/main.css

```diff
@@ -2,623 +2,650 @@ html, body {
+  color:#fff1ef;
+}
+#cta-block .inner-wrapper h3{
+  margin-bottom:15px;
+  color:#fff;
+}
+#cta-block .inner-wrapper p{
+  margin:0 auto 35px auto;padding:0;
+  line-height:100%;
+}
 
-  #who-we-are .team-photo {
-    margin: 3px 15px 0 0;
-  }
+/*FOOTER*/
+footer{
+  background:rgba(24,34,41, 1);
+  border-top:1px solid rgba(82,134,142,.3);
+  color:#52868e;
+  padding:0;
+  width:1000px;
+  margin:0 auto;
+  text-align:center;
 }
 
+#footer-wrapper{
+  background:rgba(24,34,41, 1);
+}
 
-@media screen and (max-width: 650px) {
-  #who-we-are .component-list-item {
-    margin: 10px 0 30px 0;
-  }
+footer ul{
+  margin: 35px 0 20px 0;padding:0;
+  list-style:none;
+}
 
-  #who-we-are .team-photo {
-    float: none;
-    margin: 0 0 15px 0;
-  }
+footer ul.social li{
+  margin:0;padding:0;
+  display:inline-block;
 }
 
-.component-list-wrapper {
-  display: -webkit-box;
-  display: -webkit-flex;
-  display: -ms-flexbox;
-  display: flex;
-  list-style: none;
-  margin: 0;
-  padding: 0;
-  -webkit-flex-wrap: wrap;
-      -ms-flex-wrap: wrap;
-          flex-wrap: wrap;
-  -webkit-box-pack: justify;
-    -webkit-justify-content: space-between;
-        -ms-flex-pack: justify;
-            justify-content: space-between
+footer ul.social li a{
+  color:#52868e;
+  text-decoration:none;
+  margin:0;padding:0;
+  margin: 0 10px;
 }
 
-.component-list-item {
-  width: calc(50% - 20px);
-  margin: 0 0 20px 0;
-  padding: 0 10px 0 0;
+footer .copyright{
+  margin:0;padding:0;
+  font-size:12px;
+  padding-bottom:100px;
+  line-height:140%;
+}
+footer .copyright a{
+  color:#52868e;
+  display:inline-block;
+  margin-top:5px;
 }
 
-.component-list-item-title {
-  margin: 5px 0 5px;
+footer .disclaimer-text{
+  font-size:13px;
+  letter-spacing: .02em;
+  color:#ccc;
 }
 
+/*CONTACT FORM MODAL*/
+
+#hire-us-wrapper{
+  z-index:999;
+  position:fixed;
+  top:0;
+  bottom:0;
+  left:0;
+  right:0;
+  display:flex;
+  align-items:center;
+  overflow:hidden;
+  background:rgba(15,25,30, .9);
+  display:none;
 
-#what-we-do {
-  margin-top: 60px;
 }
 
-@media screen and (max-width: 700px) {
-  #what-we-do .component-list-item {
-    width: 100%;
-  }
+#hire-us{
+  max-width:800px;
+  width:95%;
+  background:rgba(24,34,41, 1);
+  color:#fff1ef;
+  max-height:100%;
+  overflow:auto;
+  margin:5% auto;
+  padding:0 25px;
+  position:relative;
 }
 
-.social-links {
-  margin-top: 20px;
-  list-style-type: none;
+#hire-us ul{
+  list-style:none;
+  margin:0;padding:0;
+}
+
+#hire-us h3{
+  text-align:center;
+  font-size:72px;
+  margin:50px 25px 0 25px;
+  padding:0;
+  color:#fff;
+}
+#hire-us p{
+  text-align:center;
+  padding:0 100px 25px 100px;
+}
+#hire-us a{
+  color:#efefef;
+}
+
+#hire-us .modal-dismiss-btn{
+  position:absolute;
+  top:0;right:0;
+  font-size:24px;
+  padding:10px 15px;
+  font-family: "mr-eaves-xl-modern", helvetica;
+  font-weight:400;
+  color:rgba(255,255,255,.5);
+  cursor:pointer;
+}
+
+input,textarea {
```

Comment:
What about putting all tag selectors like this or those `button[disabled]` type ones up top? Putting it up top will make it easier to find the base styles to overwrite